networkx/algorithms/community/quality.py

```diff
@@ -254,6 +254,17 @@ def modularity(G, communities, weight="weight"):
     `G`, $k_i$ is the degree of $i$ and $\delta(c_i, c_j)$
     is 1 if $i$ and $j$ are in the same community and 0 otherwise.
 
+    According to [2]_ this can be reduced to
+
+    .. math:: Q = \sum_{c=1}^{n} =
+    \left[ \frac{L_c}{m} - \left( \frac{k_c}{2m} \right) ^2 \right]
+
+    where the sum iterates over all communities $c$, $m$ is the number of edges,
+    $L_c$ is the number of intra-community links for community $c$,
+    $k_c$ is the sum of degrees of the nodes in community $c$.
+
+    The second formula is the one actually used in calculation of the modularity.
+
     Parameters
     ----------
     G : NetworkX Graph
```

Comment:
Would you be willing to add the ```weight``` parameter to the docstring? The docstrings in this module all need work, but this is one of the better ones and will bring this function up to an acceptable level.

Reply:
I totally overlooked that. I've copied the description for the parameter from other similar functions and adapted it slightly.